backend/kesaseteli/applications/jobs/daily/clean_inactive_youth_applications.py

```diff
@@ -1,16 +1,21 @@
+import logging
+
 from dateutil.relativedelta import relativedelta
 from django.utils import timezone
 from django_extensions.management.jobs import DailyJob
 
 from applications.models import YouthApplication
 
+LOGGER = logging.getLogger(__name__)
 
 
 class Job(DailyJob):
-    help = "Clean all inactive youth applications older than week."
+    help = "Cleaning all inactive youth applications older than week."
 
     def execute(self):
+        LOGGER.info("Clean all inactive youth applications older than week...")
```

Comment:
Did you accidentally change help but did you mean to change execute/LOGGER.info here?

Reply:
fixed